partner_firstname/models.py

```diff
@@ -155,7 +155,12 @@ def _get_inverse_name(self, name, is_company=False):
     def _inverse_name(self):
         """Try to revert the effect of :meth:`._compute_name`."""
         parts = self._get_inverse_name(self.name, self.is_company)
-        self.lastname, self.firstname = parts["lastname"], parts["firstname"]
+        values = {
+            "lastname": parts["lastname"],
+            "firstname": parts["firstname"]
+        }
+        self.write(values)
```

Comment:
@hbrunn , I am not arguing. `write` shouldn't be there, but `update` doesn't fix the problem stated either.

Just more mental load to have when using this module.